docs/Makefile

```diff
@@ -4,7 +4,7 @@
 # You can set these variables from the command line, and also
 # from the environment for the first two.
 # -W: if there are warnings, treat them as errors and exit with status 1.
-SPHINXOPTS    ?= -W
```

Comment:
I'd ask you to please revert this change. We have good reasons for wanting the actions to fail when warning arise.